plugins/authenticator/guest-js/index.ts

```diff
@@ -2,23 +2,33 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-License-Identifier: MIT
 
+import type { invoke } from "@tauri-apps/api/primitives";
+
+/** @ignore */
 declare global {
   interface Window {
-    __TAURI_INVOKE__: <T>(cmd: string, args?: unknown) => Promise<T>;
+    __TAURI_INTERNALS__: {
+      invoke: typeof invoke;
+    };
   }
 }
 
 export class Authenticator {
   async init(): Promise<void> {
-    return await window.__TAURI_INVOKE__("plugin:authenticator|init_auth");
+    return await window.__TAURI_INTERNALS__.invoke(
+      "plugin:authenticator|init_auth",
+    );
```

Comment:
We shouldn't really on `window.__TAURI_INTERNALS__` anymore, it is internal and should be treated as such in this repo, we should use `invoke`directly from `@tauri-apps/api/primitives` which provides the abstraction around the internals.

Reply:
we do this for the plugins because otherwise each plugin global JS script would consume an extra 1.5-2KB on the binary :(

Reply:
actually I'm not seeing a huge change anymore... weird

Reply:
Maybe your cleanup is saving us :)

Reply:
pushed, but on the notification init.js script I can't really do anything about it :(

Reply:
we could turn that script into a ts file that we compile with rollup

Reply:
the things we do for love